### PR TITLE
docs/secure-connections/sauce-connect-5/operation: update kubernetes doc for new helm chart

### PR DESCRIPTION
**Pull request by mmatczuk**
This synchronizes content with the helm chart readme

https://github.com/saucelabs/helm-charts/pull/11

### CODE REVIEW

File: docs/secure-connections/sauce-connect-5/operation/kubernetes.md [hunk 1]
```diff
@@ -7,46 +7,52 @@ sidebar_label: Kubernetes
...
+helm install sauce-connect  saucelabs/sauce-connect --values /path/to/values.yaml --set sc.tunnelName=your-pool-name --set tunnelPoolSize=1
+  ```
+
+### Application logs
+
+Use the following commands in order to get the application logs:
+
+```bash
+POD_NAME=$(kubectl get pods --namespace default -l "app.kubernetes.io/name=sauce-connect,app.kubernetes.io/instance=sauce-connect" -o jsonpath="{.items[0].metadata.name}")
+kubectl logs $POD_NAME -f
```
Comment (Choraden) [inline docs/secure-connections/sauce-connect-5/operation/kubernetes.md:45]: How about getting logs with selector:
```
kubectl logs -l "app.kubernetes.io/name=sauce-connect,app.kubernetes.io/instance=sauce-connect" -f
```
This allows for getting logs from all sc pods.

Comment (mmatczuk) [reply docs/secure-connections/sauce-connect-5/operation/kubernetes.md:45]: Ofc.

File: docs/secure-connections/sauce-connect-5/operation/kubernetes.md [hunk 2]
```diff
@@ -7,46 +7,51 @@ sidebar_label: Kubernetes
...
-  user: johndoe
-  accessKey: "xxx-xxx-xxx"
-  tunnelName: "my-k8s-tunnel"
-tunnelPoolSize: 2
-terminationGracePeriodSeconds: 600
+config:
+  region: us-west
+  username: johndoe
+  access-key: "xxx-xxx-xxx"
+  tunnel-name: "my-k8s-tunnel"
```
Comment (waggledans) [inline docs/secure-connections/sauce-connect-5/operation/kubernetes.md:27]: I'd add a note about `tunnelPoolSize` and `terminationGracePeriodSeconds` here. Those 2 are important and no default is good for all users.

File: docs/secure-connections/sauce-connect-5/operation/kubernetes.md [hunk 3]
```diff
@@ -7,46 +7,51 @@ sidebar_label: Kubernetes
...
-- Run Helm install
+For more information about the configuration options, see the [sc run command reference](https://docs.saucelabs.com/dev/cli/sauce-connect-5/run).
 
-```bash title="helm install"
+#### Install the Helm chart
+
+```bash
 helm repo add saucelabs https://opensource.saucelabs.com/helm-charts
-helm install sauce-connect  saucelabs/sauce-connect --values /path/to/values.yaml
+helm install sauce-connect  saucelabs/sauce-connect --values /path/to/values.yaml --set sc.tunnelName=your-pool-name --set tunnelPoolSize=1
```
Comment (waggledans) [inline docs/secure-connections/sauce-connect-5/operation/kubernetes.md:36]: this changes to `config.tunnelName=your-pool-name`

**Comment by mmatczuk** [comment]
@waggledans PTAL, when we agree on the content I'd update it in other places.

**Review by waggledans — APPROVED** [review]

**Pull request merged**